habushu-maven-plugin/src/test/resources/specifications/package-version-override.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Poetry package version managed by Habushu
+  Habushu will automatically manage the package version within the pyproject.toml file to match that of the POM file.
+
+  Scenario Outline: SemVer 2 POM versions are translated to their PEP 440 equivalent
+    Given a Habushu module with a POM version of "<semver>"
+    When the POM version is translated to a PEP 440 version
+    Then the pyproject.toml file is updated with the version "<pep440>"
+
+    Examples:
+      | semver        | pep440       | comment        |
+      | 1.2.3         | 1.2.3        | final release  |
+      | 1.2.3-rc.4    | 1.2.3rc4     | semver 2 rc    |
+      | 1.2.3-rc4     | 1.2.3-rc4    | semver 1 rc    |
```

Comment:
I: SemVer 1.0 is actually compatible with PEP440, so no translation is necessary.